src/blob_gc_job.h

```diff
@@ -90,9 +90,10 @@ class BlobGCJob {
   uint64_t io_bytes_written_ = 0;
 
   Status DoRunGC();
-  Status BuildIterator(std::unique_ptr<BlobFileMergeIterator>* result);
-  Status DiscardEntry(const Slice& key, const BlobIndex& blob_index,
-                      bool* discardable);
+  void BatchWriteNewIndices(BlobFileBuilder::OutContexts &contexts, Status *s);
+  Status BuildIterator(std::unique_ptr<BlobFileMergeIterator> *result);
+  Status DiscardEntry(const Slice &key, const BlobIndex &blob_index,
+                      bool *discardable);
```

Comment:
format

Reply:
I've run `bash ./scripts/format-diff.sh` seems nothing happened. Need more hints.

Reply:
Not sure where went wrong. but `T &` and `T *` is clearly not in line with other functions.